inst/tests/tests.Rraw

```diff
@@ -18330,3 +18330,184 @@ if (test_bit64) {
+test(2249.25, forderv(d, c("a","b")), 2:1, output="forder.*opt=-1.*took")
+test(2249.26, forderv(d, c("a","b"), lazy=FALSE), 2:1, output="forder.*opt=0.*took")
+test(2249.27, forderv(d, c("b","a")), integer(), output="forder.*opt=1.*took")
+test(2249.28, forderv(d, c("b","a"), lazy=FALSE), integer(), output="forder.*opt=0.*took")
+options(datatable.verbose=FALSE)
+d = copy(dd)
+setindexv(d, c("a","b"))
+options(datatable.verbose=TRUE)
+test(2249.31, forderv(d, c("a","b")), integer(), output="forder.*opt=2.*took")
+test(2249.32, forderv(d, c("a","b"), lazy=FALSE), integer(), output="forder.*opt=0.*took")
+test(2249.33, forderv(d, c("b","a")), 2:1, output="forder.*opt=-1.*took")
+test(2249.34, forderv(d, c("b","a"), lazy=FALSE), 2:1, output="forder.*opt=0.*took")
+options(datatable.verbose=FALSE)
+setindexv(d, NULL)
+setindexv(d, c("b","a"))
+options(datatable.verbose=TRUE)
+test(2249.35, forderv(d, c("a","b")), integer(), output="forder.*opt=-1.*took")
+test(2249.36, forderv(d, c("a","b"), lazy=FALSE), integer(), output="forder.*opt=0.*took")
+test(2249.37, forderv(d, c("b","a")), 2:1, output="forder.*opt=2.*took")
+test(2249.38, forderv(d, c("b","a"), lazy=FALSE), 2:1, output="forder.*opt=0.*took")
+options(datatable.verbose=FALSE)
+setindexv(d, NULL)
+setindexv(d, list(c("a","b"), c("b","a")))
+options(datatable.verbose=TRUE)
+test(2249.41, forderv(d, c("a","b")), integer(), output="forder.*opt=2.*took")
+test(2249.42, forderv(d, c("a","b"), lazy=FALSE), integer(), output="forder.*opt=0.*took")
+test(2249.43, forderv(d, c("b","a")), 2:1, output="forder.*opt=2.*took")
+test(2249.44, forderv(d, c("b","a"), lazy=FALSE), 2:1, output="forder.*opt=0.*took")
+options(datatable.verbose=FALSE)
+d = copy(dd)
+setkeyv(d, c("a","b"))
+setindexv(d, list(c("a","b"), c("b","a")))
+options(datatable.verbose=TRUE)
+test(2249.51, forderv(d, c("a","b")), integer(), output="forder.*opt=1.*took", notOutput="forder.*opt=2.*took") # idxOpt is not reached
+test(2249.52, forderv(d, c("a","b"), lazy=FALSE), integer(), output="forder.*opt=0.*took")
+test(2249.53, forderv(d, c("b","a")), 2:1, output="forder.*opt=2.*took")
+test(2249.54, forderv(d, c("b","a"), lazy=FALSE), 2:1, output="forder.*opt=0.*took")
+options(datatable.verbose=FALSE)
+setkeyv(d, NULL)
+setindexv(d, list(c("a","b"), c("b","a")))
+options(datatable.verbose=TRUE)
+test(2249.55, forderv(d, c("a","b")), integer(), output="forder.*opt=2.*took", notOutput="forder.*opt=1.*took")
+test(2249.56, forderv(d, c("a","b"), lazy=FALSE), integer(), output="forder.*opt=0.*took")
+options(datatable.verbose=FALSE)
+d = copy(dd)
+setkeyv(d, c("a","b"))
+setindexv(d, list(c("a","b"), c("b","a")))
+options(datatable.verbose=TRUE)
+ab = structure(integer(), starts=1:2, maxgrpn=1L, anyna=0L, anyinfnan=0L, anynotascii=0L, anynotutf8=0L)
+ba = structure(2:1, starts=1:2, maxgrpn=1L, anyna=0L, anyinfnan=0L, anynotascii=0L, anynotutf8=0L)
+test(2249.60, forderv(d, c("a","b")), c(ab), output="forder.*opt=1.*took")
```

Comment:
sorry, i got lost here. why did `forderv()` suddenly start returning extra attributes? I don't see how this test differs from 2249.51?

Reply:
Oh, `c(ab)` strips those. Remove this test then? Or it's only here as a baseline reference for the subsequent tests?

Maybe emphasize that with a comment?